### PR TITLE
global styles migration in partner dashboard

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: app/components/partner/credit-transfer/index.scss
Comment (future-pirate-king) [inline app/components/partner/credit-transfer/index.scss:1]: can we delete this file

File: app/components/partner/registration-request-pending/index.hbs [hunk 1]
```diff
@@ -1,41 +1,59 @@
 <div local-class='request' data-test-pending-request ...attributes>
-  <div local-class='user'>
-    <div
-      local-class='email'
-      data-test-pending-request-email
-    >{{@request.email}}</div>
-    <div
-      local-class='name'
-      class='text-lightgray'
+  <AkStack @direction='column' @spacing='0.5' class='pr-2' local-class='user'>
```
Comment (future-pirate-king) [inline app/components/partner/registration-request-pending/index.hbs:2]: local class user is removed

File: app/components/partner/registration-request-rejected/index.hbs [hunk 2]
```diff
@@ -1,35 +1,53 @@
 <div local-class='request'>
-  <div local-class='user'>
-    <div local-class='email'>{{@request.email}}</div>
-    <div local-class='name' class='text-lightgray'>{{@request.fullName}}</div>
-  </div>
-  <div local-class='company'>{{@request.data.company}}</div>
-  <div
-    local-class='date'
-    class='text-lightgray'
+  <AkStack @direction='column' @spacing='0.5' class='pr-2' local-class='user'>
```
Comment (future-pirate-king) [inline app/components/partner/registration-request-rejected/index.hbs:2]: local class user is removed

File: tests/integration/components/partner/registration-request-pending-list-test.js [hunk 3]
```diff
@@ -131,10 +131,11 @@ module(
...
 
-      assert.equal(header.children[0].textContent, 'Requested by');
-      assert.equal(header.children[1].textContent, 'Company');
-      assert.equal(header.children[2].textContent, 'Requested');
-      assert.equal(header.children[3].textContent, '');
+      assert.true(header.children[0].textContent.trim().includes('Requested'));
+      assert.true(header.children[0].textContent.trim().includes('by'));
+      assert.equal(header.children[1].textContent.trim(), 'Company');
+      assert.equal(header.children[2].textContent.trim(), 'Requested');
+      assert.equal(header.children[3].textContent.trim(), '');
```
Comment (future-pirate-king) [inline tests/integration/components/partner/registration-request-pending-list-test.js:138]: use `strictEqual` instead of `equal`

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/partner/credit-transfer/index.scss
Comment (Yibaebi) [reply app/components/partner/credit-transfer/index.scss:1]: It's already deleted.

File: tests/integration/components/partner/registration-request-pending-list-test.js (hunk 3 above)
Comment (Yibaebi) [reply tests/integration/components/partner/registration-request-pending-list-test.js:138]: Done

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**